zerver/webhooks/slack/view.py

```diff
@@ -23,20 +24,39 @@ def api_slack_webhook(
     user_name: str,
     text: str,
     channel_name: str,
-    stream: str = "slack",
-    channels_map_to_topics: str = "1",
+    channels_map_to_topics: Optional[str] = None,
 ) -> HttpResponse:
-    if channels_map_to_topics not in VALID_OPTIONS.values():
-        raise JsonableError(_("Error: channels_map_to_topics parameter other than 0 or 1"))
+    content = ZULIP_MESSAGE_TEMPLATE.format(message_sender=user_name, text=text)
+    topic_name = "Message from Slack"
```

Comment:
Two things here:
- The message template doesn't have any Slack channel information, which is fine for the `channels_map_to_topics` options, but for the generic webhook handling with single stream/topic, it might be nice to instead have a message template with the channel name in it?
- The generic topic name has been marked as translated, but this is inconsistent with other webhooks, so I removed it. But maybe it would be better to keep it translated so as not to break any existing expectations for this integration?

Reply:
It's probably reasonable to just make this consistent.